packages/ui/src/ui-component/lists/FlowListView.jsx

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types'
+import { Box, Grid, Skeleton, Typography } from '@mui/material'
+import ItemCard from '@/ui-component/cards/ItemCard'
+
+const FlowListView = ({ data, images, nodeTypes, isLoading, updateFlowsApi, setError, type, onItemClick }) => {
+    return (
+        <Box sx={{ display: 'flex', flexDirection: 'column', gap: 3 }}>
+            <Grid container spacing={3}>
+                {isLoading ? (
+                    Array.from({ length: 6 }).map((_, index) => (
+                        <Grid item xs={4} sm={4} md={4} key={index}>
```

Comment:
I'd change the breakpoints here to be xs=(12} sm={6}.   If you want to keep as is for some reason, then you don't need the sm or md ones at all.   Same for the below